util/list-generators/91313x/acceptable-bots-cloudflare.src

```diff
@@ -0,0 +1,123 @@
+# This list is based on the following list:
+# https://radar.cloudflare.com/traffic/verified-bots
```

Comment:
I'm not sure, but I think integrators can't have this list in their service/products because of the license

https://creativecommons.org/licenses/by-nc/4.0/

Am I wrong?